data/registry_model/blobuploader.py

```diff
@@ -26,7 +26,7 @@
     labelnames=["region"],
 )
 pushed_bytes_total = Counter(
-    "quay_registry_pushed_bytes_total", "number of bytes pushed to the registry"
+    "quay_registry_image_pushed_estimated_bytes_total", "number of bytes pushed to the registry"
```

Comment:
This isn't estimated. This is actual (since we can track every byte uploaded)

Reply:
That's a good point -- the push isn't an estimate.